common/src/group.ts

```diff
@@ -26,8 +26,10 @@ export type Group = {
   }
   pinnedItems: { itemId: string; type: 'post' | 'contract' }[]
   bannerUrl?: string
+  privacyStatus?: PrivacyStatusType
```

Comment:
It would actually be better to not have this as optional, and add a third option for 'public'.
- It's clearer and requires fewer cases in code
- You can do firestore queries for privacyStatus === 'public' (otherwise you literally can't query for it)

To make this work you'll need a script to update all existing groups to 'public'